net-irc/ngircd/ngircd-25.ebuild

```diff
@@ -0,0 +1,110 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="An IRC server written from scratch"
+HOMEPAGE="https://ngircd.barton.de/"
+SRC_URI="https://arthur.barton.de/pub/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~x64-macos"
+IUSE="debug gnutls ident irc-plus +ipv6 libressl pam +ssl strict-rfc tcpd test zlib"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	irc-plus? ( virtual/libiconv )
+	ident? ( net-libs/libident )
+	pam? ( sys-libs/pam )
+	ssl? (
+		gnutls? ( net-libs/gnutls:= )
+		!gnutls? (
+			!libressl? ( dev-libs/openssl:0= )
+			libressl? ( dev-libs/libressl:0= )
+		)
+	)
+	tcpd? ( sys-apps/tcp-wrappers )
+	zlib? ( sys-libs/zlib )
+"
+
+BDEPEND="sys-devel/automake:1.11"
+
+DEPEND="
+	${RDEPEND}
+	acct-user/ngircd
+	acct-group/ngircd
+	test? (
+		dev-tcltk/expect
+		net-misc/netkit-telnetd
+	)
+"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-25-fix-gcc-10.patch"
+	"${FILESDIR}/${PN}-25-make-env.patch"
+)
+
+src_prepare() {
+	default
+
+	if ! use prefix; then
+		sed -i \
+			-e "s:;ServerUID = 65534:ServerUID = ngircd:" \
+			-e "s:;ServerGID = 65534:ServerGID = ngircd:" \
+			doc/sample-ngircd.conf.tmpl || die
+	fi
+
+	# Disable PAM in the config either way
+	# Users should enable it there if they want it
+	sed -i -e "s/;PAM = yes/PAM = no/" doc/sample-ngircd.conf.tmpl || die
```

Comment:
The problem is that by default, if PAM is on the system, nobody can login by default and I thought this might confuse users. There's a PAMOnly option I could change instead.

Maybe I should just explain this better in pkg_postinst?

Reply:
Sure, please do.

Reply:
I think that's best.